src/python/espressomd/constraints.py

```diff
@@ -520,9 +530,10 @@ class FlowField(_Interpolated):
     Viscous coupling to a flow field that is
     interpolated from tabulated data like
 
-    :math:`F = -\\gamma \\cdot \\left( u(r) - v \\right)`
+    :math:`\\vec{F} = -\\gamma \\cdot \\left( \\vec{u}(\\vec{r}) - \\vec{v} \\right)`
```

Comment:
If we decide to differentiate between $r$ and $x$, this should have been an $x$ in the first place.